fuse/eval/metrics/ranking/efficient_active_ranking_pairwise_model.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+from typing import Callable, List, Any
+from dataclasses import dataclass
+from collections import defaultdict
```

Comment:
Note the `isort` error.

Maybe you need to reinstall the pre-commit hooks? Something like `pre-commit install` when the repo is the current directory. If I right, you'll need to re-run the pre-commits afterwards to make sure it applies the changes :)

Reply:
Besides that the Jenkins seems to pass (after a rebuilt)